### PR TITLE
 fix HUD armor percent gauge 

### PR DESCRIPTION
**Pull request by CommonLoon102**
Related issue: https://github.com/nukeykt/NBlood/issues/106

In case of two resolutions which have 1.25 (5:4) ratio, the armor gauge was shown incorrectly, it showed about 75% instead of 100%.

### CODE REVIEW

**Comment by CommonLoon102** [comment]
This commit brought in the two new 1.25 aspect ratio resolutions:
https://github.com/nukeykt/NBlood/commit/2476526095b6d5f83378aa8e517d79bf487d8b8d

**Comment by CommonLoon102** [comment]
This fix also fixes the text align in 2D mode. Without this fix, the map name is not aligned to the right, but there is some space between the last character and the edge of the game window.

File: source/blood/src/view.cpp [hunk 1]
```diff
@@ -1597,7 +1597,8 @@ void viewInit(void)
 
 void viewResizeView(int size)
 {
-    int xdimcorrect = scale(ydim, 4, 3);
+    double ratio = (double)xdim / ydim;
+    int xdimcorrect = ratio < 1.26 && ratio > 1.24 ? scale(ydim, 5, 4) : scale(ydim, 4, 3);
```
[the remark this answers is not in the document]
Comment (nukeykt) [reply source/blood/src/view.cpp]: You can test this with any arbitrary resolution using vidmode console command:
`vidmode <xres> <yres> <bpp> <fs>`

[the remark this answers is not in the document]
Comment (CommonLoon102) [reply source/blood/src/view.cpp]: I've tested it with every resolution, both HUD sizes and the 2D mode too, and it works fine, thanks.

**Comment by nukeykt** [comment]
Thanks again :)

**Pull request merged**